### DISCUSSION TITLE
Change footer button text?

### DISCUSSION THREAD

**Discussion by swandogg** [gilbarbara/react-joyride#659] [Q&A] [answered]
Is there a quick way to change the text of a final step of a tour from 'Last' to 'Close' or 'Finish'? Wondering if there is a way to be able to change that without having to override the entire tooltip component...

Thanks!

**Comment by gilbarbara** [comment, marked as answer]
Yes, using `locale`. it is the [docs](https://docs.react-joyride.com/props).

**Reply by swandogg** [reply]
oh cool! yeah I was looking at the codebox demos and kept thinking it needed to be 
```
obj: {
  en: {
    last: 'example'
  }
}
```
which wasn't working, but just `obj: { last: 'example' }` works fine.

thanks for the reply!
